action.yml

```diff
@@ -32,6 +32,13 @@ inputs:
     description: "Boolean to determine if the built image should be pushed"
     required: false
     default: 'false'
+  cache:
+    description: "Boolean to determine if the build should use the cache"
+    required: false
+    default: 'false'
+  cache_tag:
```

Comment:
can we select smart defaults here?

can we note it's required when `cache=true`?